cilium-cli/connectivity/check/junit.go

```diff
@@ -68,7 +69,8 @@ func (j *JUnitCollector) Collect(ct *ConnectivityTest) {
 			j.testSuite.Failures++
 			msgs := []string{}
 			for _, a := range t.failedActions() {
-				msgs = append(msgs, a.String())
+				owners := ct.GetOwners(a.Scenario())
+				msgs = append(msgs, fmt.Sprintf("%s - Owners: %s", a, strings.Join(owners, ", ")))
```

Comment:
Looks OK at a glance, though if we ever want to encode more data into the `testcase.failure.data` field it might be nice to have a delimiter different from ` - `.

Reply:
Good point. What if we use `;metadata;`? Something like `fmt.Sprintf("%s;metadata;Owners: %s", a, strings.Join(owners, ", "))`

Reply:
Sounds good yeah. Lower likelihood of accidentally breaking the split logic.